CONTRIBUTING.rst

```diff
@@ -28,6 +28,32 @@ request" instead (or waiting to request reviewers).
 .. [*] You'd think this would be obvious, but far too many people ignore
        ``README``\ s in practice
 
+Summary of the review process
+===============================
+
+A detailed description can be found in a separate document about the `review
+process <https://cosmo-epfl.github.io/librascal/dev_guide/review_process.html>`_
+
+For developers
+ * We want a clean and tested master branch, which is why we review code and use
+   continuous integration tools.
+ * Contact us if you have any questions.
+ * Start the discussion about your contribution early by using the
+   Draft-Pull-Request feature of Github.
+ * Use our Pull-Request template in preparing your PR.
+ * Provide tests for the new/changed functionalities in the test suite of the library.
```

Comment:
```suggestion
 * Provide tests for the new/changed functionality in the test suite of the library.
```
"functionality" is a mass noun in this context and therefore can't be pluralized. Plus, "functionalities" just sounds kind of weird.